.travis.yml

```diff
@@ -65,3 +65,15 @@ script:
 
   # Code coverage
   - coverage report
+
+
+
+notifications:
+  slack: andela:l6HtahN5TFwe7CaKTqxjFCrJ
+
+# After the build test with coveralls
+after_success:
+- coveralls
+
+
```

Comment:
@andela-oadeniran Check out the first badge in the Readme
<img width="674" alt="screen shot 2017-02-21 at 13 21 51" src="https://cloud.githubusercontent.com/assets/9822168/23160810/f305b164-f838-11e6-9fa8-d803e12c910f.png">